### ISSUE TITLE
secret question, not showing full question on mobile.

### ISSUE THREAD

**Issue by MollieCourtenay** [healthlocker/healthlocker#287]
As a user on a mobile device, I can read the full security question on my scroller at the bottom of my screen.

Currently looks like this:
![image uploaded from ios 1](https://cloud.githubusercontent.com/assets/25012688/23917421/2c6cbc68-08e7-11e7-874f-478589ff292e.jpg)

**Comment by katbow** [comment]
I looked into this more and [there is no way to style the options in a select tag on IOS](https://stackoverflow.com/questions/12651507/styling-the-first-option-of-a-select-input-on-ios) because IOS browsers take full control of styling select list's contents. 

The only other option I found for this is to simulate the drop down with JavaScript.

Alternatively, we can limit the question length to the number of characters that can be displayed. These are currently questions I found & added in myself.
